custom_components/hacs/__init__.py

```diff
@@ -30,48 +28,33 @@
 from .data_client import HacsDataClient
 from .enums import ConfigurationType, HacsDisabledReason, HacsStage, LovelaceMode
 from .frontend import async_register_frontend
-from .utils.configuration_schema import hacs_config_combined
 from .utils.data import HacsData
-from .utils.logger import LOGGER
 from .utils.queue_manager import QueueManager
 from .utils.version import version_left_higher_or_equal_then_right
 from .websocket import async_register_websocket_commands
 
-CONFIG_SCHEMA = vol.Schema({DOMAIN: hacs_config_combined()}, extra=vol.ALLOW_EXTRA)
```

Comment:
<https://github.com/hacs/integration/blob/7df6f1dfacbcb8ba9253f7f7d9a1538b7ca6d1c6/.github/workflows/validate.yml#L143>
This action should be removed here, as this only really "test" the YAML config